pipelines/batch/src/main/java/com/google/fhir/analytics/model/BulkExportResponse.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020-2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.fhir.analytics.model;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.fhir.analytics.converter.JsonDateDeserializer;
+import com.google.fhir.analytics.converter.JsonDateSerializer;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import lombok.Data;
+
+/** The response body of the bulk export job status */
+@Data
```

Comment:
ditto

Reply:
ditto (`final`).

Reply:
Changed to `record` and used `Builder` pattern